docs/concepts/gardenlet.md

```diff
@@ -111,29 +111,6 @@ for example, using command  `kubectl certificate approve  seed-csr-<...>`).
 If that doesn’t happen within 15 minutes,
 the gardenlet repeats the process and creates another CSR.
 
-## Seed Config versus Seed Selector
```

Comment:
Can you add some explanation on how the `seedConfig` is handled, as it's missing completely now in this doc.